src/nbl/asset/utils/CJITIncludeLoader.cpp

```diff
@@ -0,0 +1,56 @@
+
+#include "CJITIncludeLoader.h"
+
+
+CJITIncludeLoader::CJITIncludeLoader()
+{
+    m_includes["nbl/builtin/hlsl/jit/device_capabilities.hlsl"] = collectDeviceCaps(limits, features);
+}
+
+std::optional<std::string> CJITIncludeLoader::getInclude(const system::path& searchPath, const std::string& includeName) const
+{
+    system::path path = searchPath / includeName;
+
+    if (searchPath == "nbl/builtin/hlsl/jit")
```

Comment:
you can pretty much assert this if you got so far

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/605#discussion_r1392646331